src/main/java/build/buildfarm/common/redis/RedisNodeHashes.java

```diff
@@ -44,7 +44,7 @@
 @Log
 public class RedisNodeHashes {
   private static final Iterable<List<List<Long>>> SINGLETON_NODE_SLOT_RANGES =
-      ImmutableList.of(ImmutableList.of(ImmutableList.of(0l, CLUSTER_HASHSLOTS - 1l)));
+      Collections.singleton(ImmutableList.of(ImmutableList.of(0l, CLUSTER_HASHSLOTS - 1l)));
```

Comment:
That rather makes the next interior ImmutableList.of a useful singleton candidate, no?

Reply:
Curiously, `Collections.singleton(...)` returns a `java.util.Set` of one item! I would not have guessed that.